src/mixins/changeStateMixin.js

```diff
@@ -77,16 +82,13 @@ export const changeStateMixin = {
       }
     },
     checkVersion() {
-      if (
-        this.flowRun.flow.core_version[0] > 0 ||
-        this.flowRun.flow.core_version[2] > 7 ||
-        (this.flowRun.flow.core_version[2] > 6 &&
-          this.flowRun.flow.core_version[4] > 2)
-      ) {
-        return true
-      } else {
-        return false
-      }
+      let split = this.flowRun?.flow?.core_version.split('.')
+
+      let major = parseInt(split[0]),
+        minor = parseInt(split[1]),
+        patch = parseInt(split[2])
+
+      return major >= 1 || (minor >= 13 && patch >= 0)
```

Comment:
is this checking for 1.13.0?

Reply:
Checking for major version 1 or minor version 13 +

Reply:
AH 👍